src/components/AnalysisContent/ContentNavigation.js

```diff
@@ -22,12 +23,12 @@ function AnalysisContentNavigation({
 }) {
   const classes = useStyles(props);
   const generateHref = index => {
-    const item = content.body[index];
-    return `#${item.id}`;
+    const item = content.topics[index];
+    return `#${item.post_name}`;
```

Comment:
I'm thinking `post_name` _may_ contain spaces and hence not very good for HTML element **id**/**href**

Reply:
@kilemensi post_name is the slug of a post created by wordpress. It has no spacing

Reply:
:while_shaking_head: Why are we calling it _name_ then? Alright @KhadijaMahanga

Reply:
@kilemensi I will give it another name, that's from wordpress :laughing: